sdk/python/pulumi_azuread/get_application_published_app_ids.py

```diff
@@ -59,7 +59,7 @@ def get_application_published_app_ids(opts: Optional[pulumi.InvokeOptions] = Non
     """
     Use this data source to discover application IDs for APIs published by Microsoft.
 
-    This data source uses an [unofficial source of application IDs](https://github.com/manicminer/hamilton/blob/main/environments/published.go), as there is currently no available official indexed source for applications or APIs published by Microsoft.
```

Comment:
Curious where this change came from.

Reply:
https://github.com/hashicorp/terraform-provider-azuread/commit/e503e815c27330cb99bc4f42741242be8e293a66 introduced a forbidden link in that sentence, so the bridge expunged the sentence.